libs/blocks/hero-marquee/hero-marquee.js

```diff
@@ -48,10 +52,32 @@ function decorateQr(el) {
   });
 }
 
-function decorateLockupFromContent(el) {
+async function loadIconography() {
+  await new Promise((resolve) => { loadStyle(`${base}/styles/iconography.css`, resolve); });
+}
+
+async function decorateLockupFromContent(el) {
   const rows = el.querySelectorAll(':scope > p');
   const firstRowImg = rows[0]?.querySelector('img');
-  if (firstRowImg) rows[0].classList.add('lockup-area');
+  if (!firstRowImg) return;
+  await loadIconography();
+  rows[0].classList.add('lockup-area');
+  rows[0].childNodes.forEach((node) => {
+    if (node.nodeType === 3 && node.nodeValue?.trim()) {
+      const newSpan = createTag('span', { class: 'lockup-label' }, node.nodeValue);
+      node.parentElement.replaceChild(newSpan, node);
+    }
+  });
+}
+
+function decorateLockupRow(el, classes) {
+  const child = el.querySelector(':scope > div');
+  loadIconography();
+  child?.classList.add('lockup-area');
+  const iconSizeClass = classes?.find((c) => c.endsWith('-icon'));
+  if (iconSizeClass) el.classList.remove(iconSizeClass);
+  const lockupSize = iconSizeClass ? `${iconSizeClass.split('-')[0]}-lockup` : 'l-lockup';
+  el.classList.add(lockupSize);
```

Comment:
Nit: I think this can be shorter
```suggestion
  el.classList.add(`${iconSizeClass?.split('-')[0] || 'l'}-lockup`);
```

Reply:
@robert-bogos  - Good eye. Love to keep it concise. I've updated/tested this and it looks good.